### DISCUSSION TITLE
Group of checkboxes should make only one selectable

### DISCUSSION THREAD

**Discussion by arsekil** [keystonejs/keystone#8414] [Feature requests]
Hello team,

I was thinking about this, if it's not already possible to achieve somehow already, but it could be great to have this options, when I use more then one checkbox in one context, make only one selectable, just like a toggle. If I select one the other becomes unselected.

What do you think of this?

**Comment by marekryb** [comment]
That's kind of how [select field](https://keystonejs.com/docs/fields/select) works, isn't it?
If you want separate bool values out of it, then you can create [virtual fields](https://keystonejs.com/docs/fields/virtual) based on select value.
